sounds.py

```diff
@@ -1,22 +1,48 @@
 from collections import namedtuple
+from dataclasses import dataclass
+from dotenv import load_dotenv
+from google.cloud import storage
 import speech_recognition as sr
-from pathlib import Path 
+from speech_recognition import AudioData
 import os
+from pathlib import Path 
 import re
+# Imports the Google Cloud client library
+# from sounds import WavInfo
 
-WavInfo = namedtuple('WavInfo', ['filename', 'label', 'sound_bytes'])
+r = sr.Recognizer()
 
-sound_samples_folder = Path().cwd() / 'sounds/samples'
-# indicate which text to compare speech to
-speech_labels = [folder_name for folder_name in os.listdir(sound_samples_folder) if os.path.isdir(sound_samples_folder / folder_name)]
-wav_filenames = []
-for speech_label in speech_labels:
-    curfolder_wavfiles = [(speech_label, wav_filename) for wav_filename in os.listdir(sound_samples_folder / speech_label) if re.match(r'^.+\.wav$', wav_filename)]
-    wav_filenames = wav_filenames + curfolder_wavfiles
+@dataclass
```

Comment:
amazing. you could tell people about why you used this and why, it's a cool modern Python feature that I don't think a lot of people know about